examples/webgl_buffergeometry_drawrange.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - buffergeometry - lines drawcalls</title>
```

Comment:
In this case you can also change the title 😊

Reply:
Yep!

Reply:
Nice catch, fixed :)